include/pstore/core/file_header.hpp

```diff
@@ -223,6 +223,7 @@ namespace pstore {
         /// This crc is used to ensure that the fields from #signature1 to #sync_name are not
         /// modified.
         std::uint32_t crc = 0;
+        std::uint32_t unused1 = 0;
 
```

Comment:
Nice fix! A static assert needs to be added later, e.g.  
PSTORE_STATIC_ASSERT (offsetof (header, unused1) == 36);